packages/template-retail-react-app/app/components/links-list/index.jsx

```diff
@@ -59,21 +75,9 @@ const LinksList = ({
                                     </Link>
                                 </ListItem>
                             ))}
-                        </HStack>
-                    ) : (
-                        links.map((link, i) => (
-                            <ListItem key={i}>
-                                <Link
-                                    to={link.href}
-                                    onClick={onLinkClick}
-                                    {...(link.styles ? link.styles : {})}
-                                >
-                                    {link.text}
-                                </Link>
-                            </ListItem>
-                        ))
+                        </List>
                     )}
-                </List>
+                </>
```

Comment:
Similarly, is it necessary to have this empty component?

Reply:
Done